__tests__/cli.test.ts

```diff
@@ -77,6 +88,7 @@ describe('CLI', () => {
   it('mimick new heartbeat with `--rich-events` flag', async () => {
     const cli = new CLIMock([
       join(FIXTURES_DIR, 'fake.journey.ts'),
+      join(FIXTURES_DIR, 'error.journey.ts'),
```

Comment:
We need to increase the timeout for this specific test.

it ('asda', callback, timeout: number), Increasing with 10 secs should do it as we now test for 2 files and also screenshots which takes more than 5 seconds.